### PR TITLE
add over example

### PR DESCRIPTION
**Pull request by knaufk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ The first twenty-four examples are being added as an advent calendar leading up
 5. [Aggregating Data](recipes/03/03_group_by.md)
 6. [Sorting Tables](recipes/08/08_order_by.md)
 
-### Aggregating over Time (Time Windows) 
+### Aggregating over Time
 1. [Aggregating Time Series Data](recipes/06/06_group_by_window.md)
 2. [Watermarks](recipes/07/07_watermarks.md)
 3. [Analyzing Sessions in Time Series Data](recipes/09/09_group_by_session_window.md)
+4. [Rolling Aggregations on Time Series Data](recipes/10/10_over.md)
```
Comment (morsapaes) [inline README.md:24]: Are we doing it again here? 😀 Repeating "Time Series Data" over and over when we have "over time" in the section title?

Comment (knaufk) [reply README.md:24]: I don't know. I am also a bit confused by the Time Series Data as kind of a synonym for tables with a time attribute.

File: recipes/10/10_over.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+OVER window aggregates compute an aggregated value for every input row over a range of ordered rows. 
+In contrast to GROUP BY aggregates, OVER aggregates do not reduce the number of result rows to a single row for every group. 
+Instead, OVER aggregates produce an aggregated value for every input row.
+
+The order needs to be defined by a [time attribute](https://docs.ververica.com/user_guide/sql_development/table_view.html#time-attributes). 
+The range of rows can be defined by a number of rows or a time interval. 
+
+In this example, we are trying to identify outliers in the `temperature_measurements` table. 
+For this, we use an `OVER` window to calculate, for each measurement, the maximum, minimum, standard deviation and average temperature across all measurements for the same city over the previous minute.
```
Comment (morsapaes) [inline recipes/10/10_over.md:15]:
```suggestion
For this, we use an `OVER` window to calculate, for each measurement, the maximum (`MAX`), minimum (`MIN`) and average (`AVG`) temperature across all measurements, as well as the standard deviation (`STDDEV`), for the same city over the previous minute. 
```

Comment (morsapaes) [reply recipes/10/10_over.md:15]: Nit pic, but I noticed that you and Seth tend to use "we" a lot. I try to phrase things using "you" instead to follow technical writing good practices. 😏

File: recipes/10/10_over.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+OVER window aggregates compute an aggregated value for every input row over a range of ordered rows. 
+In contrast to GROUP BY aggregates, OVER aggregates do not reduce the number of result rows to a single row for every group. 
+Instead, OVER aggregates produce an aggregated value for every input row.
+
+The order needs to be defined by a [time attribute](https://docs.ververica.com/user_guide/sql_development/table_view.html#time-attributes). 
+The range of rows can be defined by a number of rows or a time interval. 
+
+In this example, we are trying to identify outliers in the `temperature_measurements` table. 
+For this, we use an `OVER` window to calculate, for each measurement, the maximum, minimum, standard deviation and average temperature across all measurements for the same city over the previous minute. 
+You can imagine that a subsequent query would filter out any temperature measurement that are higher or lower than the average by more than four standard deviations.
```
Comment (morsapaes) [inline recipes/10/10_over.md:16]: Formulate as a quoted part, saying "As an exercise, you can write another query to ..."?

Comment (morsapaes) [inline recipes/10/10_over.md:16]:
```suggestion
> As an exercise, you can try to write another query to filter out any temperature measurement that are higher or lower than the average by more than four standard deviations.
```

File: recipes/10/10_over.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+```sql
+CREATE TEMPORARY TABLE temperature_measurements (
+  measurement_time TIMESTAMP(3),
+  city STRING,
+  temperature FLOAT, 
+  WATERMARK FOR measurement_time AS measurement_time - INTERVAL '15' SECONDS
+)
+-- Free text comment
+COMMENT ''
```
Comment (morsapaes) [inline recipes/10/10_over.md:28]: Doesn't need to be here.

File: recipes/10/10_over.md (hunk 2 above)
Comment (sjwiesman) [reply recipes/10/10_over.md:15]: I've heard the arguments but I still just don't understand why "you" is better than "we". "We" is inclusive, the two of us are doing something together. "You" is not, you go do that.

File: recipes/10/10_over.md (hunk 4 above)
Comment (sjwiesman) [reply recipes/10/10_over.md:28]: +1

File: recipes/10/10_over.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# 10 Rolling Aggregations on Time Series Data
+
+:bulb: This example will show how to aggregate multiple rows, which are related to the current row, via an `OVER` window. A typical use case are rolling aggregations.
```
Comment (morsapaes) [inline recipes/10/10_over.md:3]: This sounds a bit overcomplicated. What about:

"This example will show how to calculate an aggregate or cumulative value based on a group of rows using an `OVER` window."

**Review by sjwiesman — COMMENTED** [review]
Nothing to add over @morsapaes comments.

File: recipes/10/10_over.md (hunk 5 above)
Comment (morsapaes) [inline recipes/10/10_over.md:3]:
```suggestion
:bulb: This example will show how to calculate an aggregate or cumulative value based on a group of rows using an `OVER` window. A typical use case are rolling aggregations.
```

**Pull request merged**